pytest_pootle/fixtures/revision.py

```diff
@@ -11,8 +11,9 @@
 
 
 @pytest.fixture(autouse=True)
-def revision():
+def revision(clear_cache):
```

Comment:
`clear_cache` is not necessary here if you use `Revision.set(Unit.max_revision())`.
And revision is saved in `redis` cache.